src/lib/y2caasp/widgets/controller_node.rb

```diff
@@ -79,7 +79,8 @@ def validate
     private
 
       def role
-        ::Installation::SystemRole.current_role
+        # this is a role widget so a role must be selected before displaying it
+        ::Installation::SystemRole.current_role || raise("No role selected")
```

Comment:
just stylish think, but usually convention is to use for control flow `or` instead of `||`.